_style/method-invocation.md

```diff
@@ -51,7 +51,7 @@ acceptable to omit parentheses when calling `queue.size`, but not when
 calling `println()`. This convention mirrors the method declaration
 convention given above.
 
-Religiously observing this convention will *dramatically* improve code
+Religiously observing this convention will dramatically improve code
```

Comment:
The word "religiously" seems religiously religious IMHO and does not have so much to do with earthly coding standards. I suggest using e.g. "Strictly" instead of "Religiously". I like the removal of italics. That makes the text feel less dogmatic and more factual. I'd even consider cancel "dramatically", as this is kind of hand-waving.

Reply:
I will omit both "religiously" and "dramatically"